app/src/main/java/com/example/ai_stuttering_assistant/MainActivity.kt

```diff
@@ -65,8 +71,22 @@ class MainActivity : ComponentActivity() {
         Log.d("APIKey", apiKey)
         virtualAssistant = VirtualAssistant(apiKey)
         mediaPlayer = MediaPlayer()
+
+        enableEdgeToEdge()
+
+        if( android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.P ) {
+            window.attributes.layoutInDisplayCutoutMode = WindowManager.LayoutParams.LAYOUT_IN_DISPLAY_CUTOUT_MODE_SHORT_EDGES
+        }
+
```

Comment:
The `enableEdgeToEdge()` allows the app to run edge to edge (see link in main pull request details).

The `if` statement contains code that allows the app to render into the status bar when in landscape, making it a more seamless experience.